src/components/list/MetadataContactList.tsx

```diff
@@ -31,7 +32,7 @@ const MetadataContactList: React.FC<MetadataContactListProps> = ({
               <MailOutlineIcon />
             </Grid>
             <Grid item md={11} sx={{ textAlign: "left", whiteSpace: "normal" }}>
-              <Link href={`mailto:${contact.emails[0]}`}>
+              <Link href={`mailto:${email ? email : ""}`}>
```

Comment:
I just don't want any falsy data in most of the case so i use ternary operator a lot. But because almost all the data are strings actually, so both ways are all doing the same thing for these cases. Thank you for your reminding😊